docs/reference/manifest/extendedpermissions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ExtendedPermissions element in the manifest file
+description: ''
+ms.date: 03/04/2020
+localization_priority: Normal
+---
+
+# ExtendedPermissions element
+
+Defines the extended permissions the add-in needs to work. The `ExtendedPermissions` element is a child element of [VersionOverrides](versionoverrides.md).
```

Comment:
Defines the list of extended permission?

Reply:
Updated